operator/pkg/cli/kafka.go

```diff
@@ -253,24 +254,28 @@ func (kc *KafkaClient) InspectStep(pipelineStep string, offset int64, key string
 		fmt.Printf("%s\n", string(b))
 	} else {
 		for _, topic := range ki.Topics {
+			fmt.Printf("Topic: %s\n", topic.Name)
 			for _, msg := range topic.Msgs {
 				if verbose {
-					fmt.Printf("%s\t%s\t%s\t", topic.Name, msg.Key, msg.Value)
+					fmt.Printf("%s\t%s\t", msg.Key, msg.Value)
 					for k, v := range msg.Headers {
 						fmt.Printf("\t%s=%s", k, v)
 					}
 					fmt.Println("")
 				} else {
-					fmt.Printf("%s\t%s\t%s\n", topic.Name, msg.Key, msg.Value)
+					fmt.Printf("%s\t%s\n", msg.Key, msg.Value)
```

Comment:
Not sure whether we should try to truncate the value here to a max length. It's nice to see the full message, but on the other hand we have people sending tens of Kb of json as input to the model and getting large outputs as well.

Reply:
There is a truncate (`-t`) option on cmd line args that we will not return the actual payload, which perhaps can be used if the output is too large? Maybe we could rely on this option for now?